.ci/azure/linux.yml

```diff
@@ -389,17 +389,19 @@ jobs:
   - script: $(RUN_PREFIX) $(INSTALL_TEST_DIR)/ov_conditional_compilation_tests --gtest_print_time=1 --gtest_output=xml:$(INSTALL_TEST_DIR)/TEST-ConditionalCompilation.xml
     displayName: 'Conditional Compilation Tests'
 
-  - script: $(RUN_PREFIX) $(INSTALL_TEST_DIR)/paddle_tests --gtest_print_time=1 --gtest_output=xml:$(INSTALL_TEST_DIR)/TEST-PaddleTests.xml
-    displayName: 'Paddle Tests'
+  # TODO Reenable PDPD after paddlepaddle==2.5.0 with compliant protobuf is released (ticket 95904)
+  #- script: $(RUN_PREFIX) $(INSTALL_TEST_DIR)/paddle_tests --gtest_print_time=1 --gtest_output=xml:$(INSTALL_TEST_DIR)/TEST-PaddleTests.xml
+  #  displayName: 'Paddle Tests'
```

Comment:
Now we lose PDPD unit-tests. 
@ceciliapeng2011, @wenjiew, are we ready for this?

Reply:
we have no choice

Reply:
Why is it so urgent to update ONNX and Protobuf for the sake of loss of unit-tests?

Reply:
python 3.11 support

Reply:
It's has been already discussed on various of meeting. Please focus on TF related changes.